src/DssSpell.sol

```diff
@@ -49,82 +127,196 @@ contract DssSpellAction is DssAction {
     //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
 
-    // --- Rates ---
-    // uint256 internal constant SEVEN_PT_FIVE_PERCENT_RATE = 1000000002293273137447730714;
+    function actions() public override {
+        // ----------------- Compound v2 D3M Onboarding -----------------
+        // https://vote.makerdao.com/polling/QmWYfgY2#poll-detail
```

Comment:
```suggestion
        // Poll: https://vote.makerdao.com/polling/QmWYfgY2#poll-detail
```
To match consistency with below

Reply:
I think it's fine as-is. :) Would like to stick to bugs and structural stuff so I'm going to skip these comment formatting ones.